hack/.linted_packages

```diff
@@ -10,10 +10,10 @@ cmd/kube-apiserver/app
 cmd/kube-apiserver/app/options
 cmd/kube-controller-manager
 cmd/kube-controller-manager/app/options
-cmd/kubectl
 cmd/kube-dns
-cmd/kubelet
 cmd/kube-proxy
+cmd/kubectl
+cmd/kubelet
```

Comment:
I think sorting this file in this PR is going to give you extra rebases and I wouldn't do it :)